README.md

```diff
@@ -1,23 +1,118 @@
 # Kuri EDU
-A minimal kuri robot that exposes kuri's subsystems without any autonomous behavior
+This repository contains software and activities intended to be used for
+educational purposes.  The kuri_edu folder contains a ROS package that provides 
+a minimal running robot framework and the activities folder contains ideas and
+instructions for extending the robot behavior
```

Comment:
* Either here or elsewhere in the main README, you should mention that this assumes a GBZ or EDU binary is already installed. In other words, that the `kuri_edu` package is not self-sufficient. It won't build on an arbitrary computer.
* Maybe tell them that the `kuri_edu` package is already installed in `/opt/edu` along with its dependencies. And that later on they'll be checking out the source code and modifying it.

Reply:
Done below